docs/full/README.md

```diff
@@ -228,12 +228,46 @@ reason. Below is a brief summary of these commands
 
 `version`: provides basic version information about the plugin, likely the only common command of broad use to the CLI user.
 
-`info`: provides metadata about the plugin that the CLI will use when presenting information about plugins or when performing lifecycle operations on them.
+`info`: provides metadata about the plugin that the CLI will use when presenting information about plugins or when
```

Comment:
nit: when wrapping, let's wrap consistently to 80 chars for L231-243

Reply:
updated.